### ISSUE TITLE
broken wiki links

### ISSUE THREAD

**Issue by alexban011** [yairm210/Unciv#7179]
So I noticed a link was broken so I started clicking on every link I found in the wiki. These are the broken links I've found.

<pre>     Modders >   Mods > Mod components >  the base json files
                                        the base image files
                Making a new Civilization > Get your Civ icon > NationIcons folder
                                            Adding unique units > UnitIcons
                                            Adding unique buildings > BuildingIcons
                
    Developers >    Building Locally >  Without Android Studio (doesn't work)
                    Project structure > Translations > translations templates</pre>

**Comment by alexban011** [comment]
yeah well I deleted it on github, thats where I can't figure out how to get it back :))

**Comment by SomeTroglodyte** [comment]
Any fork's branch can be fetched into your project using Studio's git UI (under any name you choose), and can subsequently be pushed to your repo - and the menus are named 'checkout', so Timo is right, and `git clone` wanting to create a new project isn't surprising either.

But wiki changes are nowadays done in master/docs like #7182 demonstrates.
